liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/InstallFeatureMojo.java

```diff
@@ -49,11 +50,20 @@ protected void doExecute() throws Exception {
     }
 
     private void installFeatures() throws PluginExecutionException {
-          
-        Set<String> featuresToInstall = getInstalledFeatures();
+        List<ProductProperties> propertiesList = InstallFeatureUtil.loadProperties(installDirectory);
+        String openLibertyVersion = InstallFeatureUtil.getOpenLibertyVersion(propertiesList);
+
+        Boolean skipBetaInstallFeatureWarning = Boolean.parseBoolean(System.getProperty("skipBetaInstallFeatureWarning"));
```

Comment:
If `skipBetaInstallFeatureWarning` can be defined as a public static constant in InstallFeatureUtil in ci.common, you can use that constant here as well.

Reply:
This variable can be a primitive `boolean` instead of `Boolean`, since `Boolean.parseBoolean` returns a `boolean`